web_search_nospace/__openerp__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Search no space",
+    'author': "Therp BV, Odoo Community Association (OCA)",
```

Comment:
@daramousk Add a `website` key as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)